pacehrh/R/pace_config.R

```diff
@@ -209,3 +244,63 @@ SetRoundingLaw <- function(value = NULL){
 
   return(invisible(prevValue))
 }
+
+#' Turn Stochasticity On/Off
+#'
+#' Debugging function to turn off all Monte Carlo variation. With stochasticity
+#' off, every experiment based on the same configuration produces the
+#' same output.
+
+#' By default the system is set up with stochasticity on.
+#'
+#' @param value Desired stochasticity. (Calling \code{SetStochasiticity()} with
+#' no parameters or value = NULL returns the current state.)
+#'
+#' @return Previous stochasticity flag value (invisible)
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' pacehrh::SetStochasticity(FALSE) # Turn off stochasticity
+#' pacehrh::SetStochasticity() # Return current stochasticity state
+#' }
+SetStochasticity <- function(value = NULL){
+  prevValue <- GPE$stochasticity
+
+  if (!is.null(value)){
+    if (rlang::is_logical(value)){
+      GPE$stochasticity <- value
+    } else {
+      traceMessage(paste0(value, " is not an allowed stochasticity flag value"))
+    }
+  }
+
+  return(invisible(prevValue))
+}
+
+#' Turn Per-Age Statistics On/Off
```

Comment:
Can we possible change the "On/Off" to "Annual/Monthly/Off"? My impression is that when using .ComputePerAgeTaskTime, the .computeAnnualTaskTimes still can finish rather quickly, but .computeMonthlyTaskTimes takes significantly longer. The three options switch would allow the user to still have annual per age task times results for some basic analyses.

Reply:
@rhanIDM : Could you move this request into a separate issue so we can close out this PR? This is a worthy change, but moving from a simple TRUE/FALSE check to a three-way selection will be a little bit tricky.

Reply:
Moving this comment to issue #235.